files/scripts/radv.sh

```diff
@@ -1,84 +0,0 @@
-#!/bin/bash
```

Comment:
Do you know why GIT shows that original `radv.sh` was deleted? Is it because the new script has new permissions (chmod permissions)? 

In the same regard, does other `sh` scripts introduced here have `executable` permissions? I think that's necessary.

Reply:
All have executable permissions. radv.sh and service_mgmt.sh are not the same.

Reply:
I understand that they are different files. My minor concern was why GIT shows that original `radv.sh` was deleted. This usually happens when contents are replaced and permissions are changed.

![image](https://user-images.githubusercontent.com/22228981/163433605-81b20dc2-b549-41e8-9a10-bcc753c5173e.png)

Reply:
@vaibhavhd The content was changed, please check the line SERVICE="radv" and SERVICE="${SCRIPT_NAME%.*}" in service_mgmt.sh